internal_transfer_with_agreed_amount/__manifest__.py

```diff
@@ -3,7 +3,7 @@
     "version": "12.0.1.0.0",
     "author": "Vauxoo",
     "website": "http://www.vauxoo.com/",
-    "license": "LGPL-3",
+    "license": "OPL-1",
```

Comment:
Remove this commit. This is open software (hosted on a public repository), it's expected to use a free license.

Reply:
Commit removed.